source/documentation/metrics-install-guide/Upgrading_Metrics_Store.html.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Replacing fluentd with Rsyslog
+---
+
+# Chapter 5: Upgrading Metrics from 4.2
+
+Since version 4.3 Rsyslog is used to gather metrics from oVirt hosts (replacing fluentd).
+
+Rsyslog is used to collect Engine, VDSM logs and the Collectd metrics, parse the data and add additional metadata and send the data to the remote metrics store.
```

Comment:
@eslutsky Rsyslog is used to collect and parse Engine, VDSM logs and Collectd metrics, add additional metadata and send the data to the remote metrics store.